verilog/analysis/verilog_linter_configuration_test.cc

```diff
@@ -771,6 +771,20 @@ TEST(RuleBundleTest, ParseRuleBundleReject) {
   EXPECT_EQ(error, absl::StrCat(kInvalidFlagMessage, " \"bad-flag\""));
 }
 
+TEST(RuleBundleTest, ParseRuleBundleAcceptGoodRulesEvenWhenRejecting) {
+  constexpr absl::string_view text = "test-rule-unknown-rules\ntest-rule-1";
+  {
+    RuleBundle bundle;
+    std::string error;
+    bool success = bundle.ParseConfiguration(text, '\n', &error);
+    ASSERT_TRUE(!success) << error;
+    ASSERT_NE(error.find(std::string(kInvalidFlagMessage)), std::string::npos)
```

Comment:
These string comparisons to look for a particular message might be reading more natrually by using the test framework HasSubstr() comparison:

```
EXPECT_THAT(error, HasSubstr(kInvalidFlagMessage));
```

(and somewhere towards the top of the file `using testing::HasSubstr;`)

Reply:
I didn't know those functions existed, thanks for the suggestion, it definitely looks better. I went with the full name rather than doing `using testing...`